Add `osu! Vintage Map Tourney!` tournament articles (#5959)
ppy/osu-wiki: wiki/Tournaments/OVMT/1/en.md

---
tags:
  - OVMT
  - OVMT1
---

# osu! Vintage Map Tourney!

The **osu! Vintage Map Tourney!** (***OVMT***) was a 1v1 single-elimination osu! tournament hosted by ![][flag_GB] [mmi-](https://osu.ppy.sh/users/5444914). It was the first instalment of osu! Vintage Map Tourney!.

## Tournament schedule

| Event | Timestamp |
| --: | :-- |
| Registration phase | 2016-07-14/2016-08-31 |
| Round of 64 | 2016-09-01/2016-09-08 |
| Round of 32 | 2016-09-09/2016-09-15 |
| Round of 16 | 2016-09-16/2016-09-19 |
| Quarterfinals | 2016-09-23/2016-09-26 |
| Semifinals & Finals | 2016-10-02/2016-10-03 |

## Prizes

| Placing | Prize(s) |
| :-: | :-- |

> | ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 1 year of osu!supporter, name engraved trophy |

Reviewer: ```suggestion
| ![Gold crown](/wiki/shared/crown-gold.png "1st place") | 1 year of osu!supporter, name-engraved trophy |
```

| ![Silver crown](/wiki/shared/crown-silver.png "2nd place") | 1 year of osu!supporter |
| ![Bronze crown](/wiki/shared/crown-bronze.png "3rd place") | 6 months of osu!supporter |

## Organisation

> osu! Vintage Map Tourney! was run by various community members.

Reviewer: ```suggestion
The osu! Vintage Map Tourney! was run by various community members.
```

- TieBreaker
  - **[Blind Stare - Shotgun Symphony (awp) [Insane]](https://osu.ppy.sh/beatmapsets/343#osu/3290)**

### Round of 64

- NoMod
  - [Alestorm - Keelhauled (awp) [Hard]](https://osu.ppy.sh/beatmapsets/9215#osu/37055)
  - [Gunther - Ding Dong Song (osuplayer111) [oSuPlaYeR mIX]](https://osu.ppy.sh/beatmapsets/5709#osu/27447)
  - [Hatsune Miku - Still Alive (James) [Hard]](https://osu.ppy.sh/beatmapsets/2194#osu/19029)
  - [NicoNicoDouga - Cirno de Gozaimasu! (Mafiamaster) [Hard]](https://osu.ppy.sh/beatmapsets/3367#osu/21577)
  - [Queen - Don't Stop Me Now (m980) [Hard ROCK!]](https://osu.ppy.sh/beatmapsets/5381#osu/26547)
- FreeMod
  - [Chatmonchy - Make Up! Make Up! (peppy) [Hard]](https://osu.ppy.sh/beatmapsets/184#osu/8708)
  - [Foo Fighters - The Pretender (Doomsday93) [Hard]](https://osu.ppy.sh/beatmapsets/5648#osu/27339)
  - [Kenji Ninuma - DISCO PRINCE (peppy) [Normal]](https://osu.ppy.sh/beatmapsets/1#osu/75)
  - [Scatman John - Scatman (Extor) [Insane]](https://osu.ppy.sh/beatmapsets/18#osu/80)
- TieBreaker
  - **[Chemistry meets m-flo - Now Or Never (Behonkiss) [Insane]](https://osu.ppy.sh/beatmapsets/952#osu/10880)**

## Match results

### Finals

> #### Grand Final

Reviewer: ```suggestion
#### Grand Finals
```

Sunday, 2 October 2016:

| Player 1 | | | Player 2 |
| --: | :-: | :-: | :-- |
| Catgirl ![][flag_US] | 1 | **4** | ![][flag_GB] **Pelzio** |

> #### 3rd Place

Reviewer: ```suggestion
#### Third place match
```

| TropicalX ![][flag_GB] | 0 | **3** | ![][flag_FI] **Sanakirja** |  |
| **BitComet** ![][flag_FI] | **3** | 2 | ![][flag_GB] mjozog3 |  |
| **at-cay** ![][flag_US] | **3** | 0 | ![][flag_GB] Dawnsday |  |
| nahh ![][flag_GB] | 0 | **1** | ![][flag_US] **Kradfiz** | *forfeit* |
| **Kosmit** ![][flag_PL] | **3** | 0 | ![][flag_SG] handsome |  |
| **Lefafel** ![][flag_FI] | **3** | 2 | ![][flag_GB] Tattoos |  |
| **Doughy** ![][flag_GB] | **3** | 1 | ![][flag_GB] NotToBlame |  |
| M-O-P-S-I ![][flag_FI] | 1 | **3** | ![][flag_SE] **Nitroz** |  |
| Hasert ![][flag_MY] | 0 | **3** | ![][flag_GB] **Doomsday is Bad** |  |
| **Muuki** ![][flag_FI] | **3** | 2 | ![][flag_US] Squilly |  |
| **Pelzio** ![][flag_GB] | **3** | 0 | ![][flag_AU] TheJimnebob |  |
| **- Auto -** ![][flag_CA] | **3** | 0 | ![][flag_GB] Khunee |  |
| **Babo-ya** ![][flag_FI] | **3** | 0 | ![][flag_ES] TriFerlol |  |
| **Halfslashed** ![][flag_US] | **3** | 0 | ![][flag_PL] \[-KuleK-\] |  |
| **dong** ![][flag_GB] | **3** | 1 | ![][flag_US] YouKittenMe |  |

## Ruleset

- This tournament is a 1v1 single-elimination international osu! tournament.
- This tournament will consist of 64 players.

> - Your rank must be lower than #100,000 to play. Signups under this rank will not be considered.

Reviewer: is the wording on these sentences correct? currently either ambiguous or confusing

```suggestion
- Your rank must be lower than #100,000 to play. Sign-ups under this rank will not be considered.
```

Author: It was the wording that was used at the time 🤷

Author: ```suggestion
- Your rank must be better than #100,000 to play. Sign-ups worse this rank will not be considered.
```
This could work and is essentially in line with how OWC words their rank rule,

Reviewer: seems ok

- With the exception of 3 maps, every map is 2009 or older.
- With the exception of 2 maps, every map is AR8 or lower.
- ScoreV1 will be used.
- If you are unable to make the scheduled match time, please contact a member of the team to arrange an alternative match time.

### Match info

- Each pool will consist of 5 NoMod, 4 FreeMod and a TieBreaker map.
  - On NoMod maps, both players must not use any mod.
  - On FreeMod maps, players are free to choose wether to use a mod or not. Every ranked mod is allowed.
    - If both players agree, DT or HT can be enabled for the map.
  - The TieBreaker map follows FreeMod rules.
- The Round of 32 is Best of 5, every round afterwards is Best of 5.
- Both players will be contacted by the match referee before the match starts in bancho chat. Players have until 15 minutes after the scheduled start time of the match to show up, or they automatically forfeit.

> - Players are able to play a warm up map if they so choose. Warmup maps should be under 5 minutes long and they must be submitted to osu.ppy.sh.

Reviewer: ```suggestion
- Players are able to play a warm-up map if they so choose. Warm-up maps should be under 5 minutes long and they must be submitted to osu.ppy.sh.
```
